.golangci.yml

```diff
@@ -149,6 +149,11 @@ issues:
     - stylecheck
     text: "ST1016: methods on the same type should have the same receiver name"
     path: .*(api|types)\/.*\/conversion.*\.go$
+  # hack/tools
+  - linters:
+    - typecheck
+    text: import (".+") is a program, not an importable package
+    path: ^tools\.go$
```

Comment:
nit: should we match the entire path and not only the filename?

Reply:
We can't, the path is relative to where golanci-lint runs, which is the tools folder in this case. It's not ideal but that's also why I added a stricter validation here

Reply:
Oh yeah makes sense